diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/FileSizeWarnTest.kt

```diff
@@ -116,4 +121,17 @@ class FileSizeWarnTest {
         file.writeText("//hello \n".repeat(2000))
         return 2000
     }
+
+    @Test
+    fun `ignoring two out of three folders`(){
+        var path = javaClass.classLoader.getResource("test/paragraph3/src/main/A/FileSizeA.kt")
+        var file = File(path!!.file)
+        lintMethod(FileSize(), file.absolutePath, file.readText(), rulesConfigList = rulesConfigListTwoIgnoreFolders)
+        path = javaClass.classLoader.getResource("test/paragraph3/src/main/B/FileSizeB.kt")
+        file = File(path!!.file)
+        lintMethod(FileSize(), file.absolutePath, file.readText(), rulesConfigList = rulesConfigListTwoIgnoreFolders)
+        path = javaClass.classLoader.getResource("test/paragraph3/src/main/C/FileSizeC.kt")
+        file = File(path!!.file)
+        lintMethod(FileSize(), file.absolutePath, file.readText(), rulesConfigList = rulesConfigListTwoIgnoreFolders)
```

Comment:
I suggest making file C the same length as A and B and checking corresponding LintError. This way it will be more clear that C is checked and not skipped as A and B.